### PR TITLE
The Slutcurity Update. Powered by Horny™

### PR DESCRIPTION
**Pull request by TotallyNotADog**
## About The Pull Request
Gives security a brand-new implant: the slave implant. It functions vaguely like a mindshield implant, where it will clear any previous brainwashing on application and replace them with an objective to be loyal to security. The implant, however, will not block further brainwashing attempts to keep the normal mindshield implant relevant. A mindshield implant will also remove the effects of this implant.

To respect OOC prefs, the slave objective will tell the victim to be sec's pet, unless the victim's lewdchem preference is set to disabled.

TODO:
- [x] Add implant with basic functionality.
- [x] Make sure implant can't be used on people who are part of sec.
- [x] Add lewd sec clothing. (e.g. slingsuit/recolored stripper suit, collars, etc.)
- [x] Add more general slave items. (e.g. slave pet bed, riding crop reskin of bdsm whip, etc.)

Wishlist:
- [ ] Slave job as roundstart. (yeah i know prisoner could be the same thing but think of the sex!)
- [ ] Special commands/trigger phrases for pets.
- [ ] Count sec slaves as actual sec officers??? (more of a policy thing to ask Dahl/Quote)

### Holy shit this got way bigger than I originally intended.
Let's go over what this contains now:
- The Slave Implant: explained above.
- A fuckton of new clothing items for slutsec. Notable items nested here:
  - Sluthailer: a sechailer with pinker, lewder phrases, and a pink paintjob to boot. Also functions as a muzzle, silencing the wearer.
  - Slutbelt: a secbelt with a pink color scheme, and can carry some lewder equipment like dildos and condoms.
  - Security pet collars: reskins of the locked pet collar and shock collar to be more in line with sec's colors.
- A couple of other miscellaneous items:
  - Riding crop: just a kinkier reskin of the BDSM whip.
  - Security pet bed: just a pet bed in security's colors for obvious reasons.
  - The Bitch Breaker
- A few smaller fixes:
  - The corset had the same issue that I encountered while making my skirt where it wouldn't appear on nonhumans. That's no longer happening now.
  - ~~The muzzle and everything that stems on it (like the ballgag) can no longer be removed by yourself! Apparently whoever put them in thought that attack_paw() was a viable replacement for attack_hand().~~
  - Apparently attack_paw() is there on purpose so that monkies can't take off the muzzle. But if monkies can't slip out of it, why can people? In any case, it's lewder if you can't take it off, so it's staying. Something something keep your prisoners quiet.

The implant can be found in security's lathe once subdermal implants is researched, and the other items can be found in the premium section of the secdrobe and the contraband section of the sectech.

#### How about some pictures?
image
image
image
image

## Why It's Good For The Game
~~Adds an alternative method of permanent incarceration to be chosen instead of permabrig for sec officers that want free labor or their dicks sucked.~~
Most of the other departments on the station have at least *some* form of unique lewdness they can engage in. Medbay has fermichems and augmentation surgery, science has their slutty borgs, cargo can order kinky stuff, engineering can build strip clubs and the like, and service is, well, service. Security has been missing out until now. Create slaves and dress them up in kinky outfits, or just dress yourself up and make arrests in a kevlar bikini. And also,

sex!

## Changelog
:cl:
add: Adds the slave implant, an all-new method of permanent incarceration while keeping the prisoner "free"!
add: Many many lewd security clothes.
add: A few kinky pieces of gear for slutcurity.
add: The muzzle (and any of its derivatives) are now no longer able to be removed alone! Better call a friend!
fix: The corset is no longer broken on nonhumans.
/:cl:

### CODE REVIEW

**Comment by Moltovz** [comment]
Oh boy time to have LOOC: no on my notepad!

**Comment by Dahlular** [comment]
jackboots but LOOOONNNGGG

**Review by Moltovz — COMMENTED** [review]
> jackboots but LOOOONNNGGG

Dahl no

File: code/game/objects/items/implants/implant_slave.dm [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		if(!target.mind)
+			ADD_TRAIT(target, TRAIT_MINDSHIELD, "implant")
+			target.sec_hud_set_implants()
+			return TRUE
+		Can't be a pet without having a mind! */
+
+		if(target.mind.has_antag_datum(/datum/antagonist/brainwashed))
+			target.mind.remove_antag_datum(/datum/antagonist/brainwashed)
+
+		if(target.mind.has_antag_datum(/datum/antagonist/rev/head) || target.mind.unconvertable || target.mind.has_antag_datum(/datum/antagonist/gang/boss) || target.mind.assigned_role == "Security Officer" || target.mind.assigned_role == "Detective" || target.mind.assigned_role == "Warden" || target.mind.assigned_role == "Head of Security")
```
[the remark this answers is not in the document]
Comment (TotallyNotADog) [reply code/game/objects/items/implants/implant_slave.dm:32]: Lawyers don't get security jumpsuits to begin with, so no.

**Review by Dahlular — APPROVED** [review]
One of our biggest PRs, and it's about sex. I couldn't be more proud

**Pull request merged**